src/contentScript.ts

```diff
@@ -36,32 +33,6 @@ type BeaconMessage =
     };
 type BeaconPageMessage = BeaconMessage | { message: BeaconMessage; sender: { id: string } };
 
-// Prevents the script from running in an Iframe
-if (window.frameElement === null) {
-  browser.storage.local.get(WEBSITES_ANALYTICS_ENABLED).then(storage => {
-    if (storage[WEBSITES_ANALYTICS_ENABLED]) {
-      let oldHref = '';
-
-      const trackUrlChange = () => {
-        const newHref = window.parent.location.href;
-        if (oldHref !== newHref) {
-          oldHref = newHref;
-
-          browser.runtime.sendMessage({
-            type: ContentScriptType.ExternalLinksActivity,
-            url: newHref
-          });
-        }
-      };
-
-      trackUrlChange();
-
-      // Track url changes without page reload
-      setInterval(trackUrlChange, TRACK_URL_CHANGE_INTERVAL);
-    }
-  });
-}
-
```

Comment:
why did you delete this?

Reply:
As I understood, the task was to send events exactly for the sites where the ads have been successfully replaced. That's why sending messages of type `ContentScriptType.ExternalLinksActivity` was moved to `src/replaceAds.tsx`

Reply:
i thought we would need to track all of these sites from the list as before, but on the sites that stream ads this ads should be replaced by slise xyz